extensions/2.0/Vendor/GRIFFEL_bim_data/schema/GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,105 @@
+            "description": "Node type and instance properties map. Used only when data is saved in a separate file."
+        },
+        "propertyValues": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "description": "Property value.",
+                "minLength": 1
+            },
+            "minItems": 1,
+            "description": "Collection of unique property values."
+        },
+        "propertyNames": {
+            "type": "array",
+            "items": {
+                "type": "string",
+                "description": "Property name.",
+                "minLength": 1
+            },
+            "minItems": 1,
+            "description": "Collection of unique property names."
+        },
+        "properties": {
+            "type": "array",
+            "items": {
+                "title": "Node Property",
+                "type": "object",
+                "description": "Key value pair - unique property (instance or type) attached to nodes. Name and value are referenced by index of corresponding root level collection.",
+                "properties": {
+                    "name": {
+                        "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                        "description": "Index of a property name in the root level collection."
+                    },
+                    "value": {
+                        "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                        "description": "Index of a property value in the root level collection."
+                    }
+                },
+                "required": [ "name", "value" ]
+            },
+            "minItems": 1,
+            "description": "Collection of unique property name - property value pairs."
+        },
+        "types": {
+            "type": "array",
+            "items": {
+                "title": "Node Type",
+                "type": "object",
+                "description": "Set of properties which are common for many nodes.",
+                "properties": {
+                    "name": {
+                        "type":"string",
+                        "description": "Name of the type."
+                    },
+                    "properties": {
+                        "type": "array",
+                        "item": {
+                            "allOf": [ { "$ref": "glTFid.schema.json" } ],
+                            "description": "Index of a property in the root level collection."
+                        },
+                        "description": "Collection of indices which point to corresponding properties of the type.",
+                        "minItems": 1
+                    }
+                },
+                "required": [ "properties" ]
+            },
+            "minItems": 1,
+            "description": "Collection of types - common sets of properties for many nodes."
+        }
+    },
+    "required": [ "node", "properties" ]
```

Comment:
This should likely be `[ "nodeProperties", "propertyValues", "propertyNames", "properties" ]`.